src/utils/validate/numberInRange.ts

```diff
@@ -4,6 +4,22 @@
  * - a number
  * - in a provided range (inclusive)
  */
+/*
```

Comment:
minor, non blocking: I believe the copyright should be before the other comment?

Reply:
Good catch, fixed this one by hand and logged a bug for myself on the plugin repo